### ISSUE TITLE
Mentor Catapult HLS

### ISSUE THREAD

**Issue by nhanvtran** [hls-fpga-machine-learning/hls4ml#149]

**Comment by danielemoro** [comment]
Hi! Is support for Catapult still being worked on? I noticed that the [HLS4ML paper](https://arxiv.org/pdf/2103.05579.pdf) specifically mentions Catapult support in Figure 8 and in the ASIC workflow section. 

@vloncar

**Comment by vloncar** [comment]
Yes! We hope to merge it soon into the main branch. The latest development is in https://github.com/dgburnette/hls4ml

**Comment by danielemoro** [comment]
Fantastic news! Thank you
